src/main/target/CHEBUZZF3/config.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <platform.h>
+
+#include "config/config_master.h"
+#include "config/feature.h"
+
+void targetValidateConfiguration(master_t *config)
+{
+	UNUSED(config);
+
+    if (featureConfigured(FEATURE_RX_PARALLEL_PWM)) {
+        // led strip needs the same ports
+        featureClear(FEATURE_LED_STRIP);
```

Comment:
I think this should be removed. With resource management, users can move the LED_STRIP to another pin, so no reason to disable it really.